orchagent/aclorch.h

```diff
@@ -373,6 +403,10 @@ class AclTable {
     bool clear();
     // Update table subject to changes
     void update(SubjectType, void *);
+
+private:
+    sai_object_id_t m_oid;
+    AclOrch *m_pAclOrch;
 };
 
 class AclOrch : public Orch, public Observer
```

Comment:
consider having an api to lock `m_countersMutex` before any operation with `AclOrch`